plugin/fetch_linux.go

```diff
@@ -200,8 +200,11 @@ func withFetchProgress(cs content.Store, out progress.Output, ref reference.Name
 		switch desc.MediaType {
 		case ocispec.MediaTypeImageManifest, images.MediaTypeDockerSchema2Manifest:
 			tn := reference.TagNameOnly(ref)
-			tagged := tn.(reference.Tagged)
-			progress.Messagef(out, tagged.Tag(), "Pulling from %s", reference.FamiliarName(ref))
+			var tag string
+			if tagged, ok := tn.(reference.Tagged); ok {
+				tag = tagged.Tag()
+			}
+			progress.Messagef(out, tag, "Pulling from %s", reference.FamiliarName(ref))
```

Comment:
Perhaps not a strict blocker, but wondering if `tag` should get some other content if it's not a tag. Comparing this to the `image pull` code, I see that uses a `tagOrDigest` variable; https://github.com/moby/moby/blob/0616b4190e0d49dab2936899947d47f4495477b1/daemon/containerd/image_pull.go#L122-L128


I think it would make sense to do the same here, WDYT?

Reply:
All for keeping consistency here! I'll change it.

Reply:
thx! Yes, also trying to prevent confusion; there's some (sometimes subtle) differences between plugins and "regular" images, so trying to prevent someone from considering this was one of those 😅